co-log/src/Colog/Monad.hs

```diff
@@ -65,6 +66,9 @@ withLog = local . overLogAction
 liftLogAction :: (Monad m, MonadTrans t) => LogAction m msg -> LogAction (t m) msg
 liftLogAction (LogAction action) = LogAction (lift . action)
 
+liftLogIO :: MonadIO m => LogAction IO msg -> LogAction m msg
+liftLogIO (LogAction action) = LogAction (liftIO . action)
```

Comment:
@cronokirby Thanks for your contribution! The reason for this function to be in `co-log-core` package is because it doesn't actually brings new dependencies. `MonadIO` and `liftIO` are already in `base` package. So it's completely okay to put this function to `co-log-core` package (and bump version of `co-log-core` to `0.1.1`). `liftLogAction` on the hand requires `transformers` package because of `MonadTrans` constraint

Reply:
Ah, yeah that makes sense